library/std/src/collections/mod.rs

```diff
@@ -252,7 +252,9 @@
 //!
 //! Several other collection methods also return iterators to yield a sequence
 //! of results but avoid allocating an entire collection to store the result in.
-//! This provides maximum flexibility as `collect` or `extend` can be called to
+//! This provides maximum flexibility as `collect` or
+//! [`extend`](https://doc.rust-lang.org/stable/std/iter/trait.Extend.html)
```

Comment:
We use a feature called intra-doc links rather than absolute urls. For example:

```suggestion
//! [`extend`][crate::iter::Extend]
```

Reply:
Oh, that's how one qualify these internal links. I should had guessed it. X) Thanks!